server/api/users/routes/get_users.js

```diff
@@ -30,7 +30,7 @@ module.exports = (server) => ({
       page: page || 0,
       include_totals: true,
       fields: 'user_id,name,email,identities,picture,last_login,logins_count,multifactor,blocked',
-      search_engine: 'v2'
+      search_engine: 'v3'
```

Comment:
this value should be taken from configuration (for example, `v2` is still valid in appliance)